drivers/led/Kconfig.gpio

```diff
@@ -7,3 +7,23 @@ config LED_GPIO
 	depends on GPIO && DT_HAS_GPIO_LEDS_ENABLED
 	help
 	  Enable driver for GPIO LEDs.
+
+if LED_GPIO
+
+config LED_GPIO_BLINK
+	bool "Software LED blinking"
+	default n
+	help
+	  Enable software LED blinking.
+
+if LED_GPIO_BLINK
+
+config LED_GPIO_BLINK_NUM
+	int "Number of LEDs to manage"
+	default 4
+	help
+	  Number of LEDs to manage.
```

Comment:
you now the number of children at compile time, so why not use that information?

Reply:
How can I retrieve it?
I can't use `ARRAY_SIZE(gpio_dt_spec_##i)` in `struct led_gpio_data`.

Reply:
I found an example in `drivers/led/lp503x.c` to do that.
Thanks for the comment.

Reply:
Fixed